nuxt/components/form/event/FormEvent.vue

```diff
@@ -552,15 +552,15 @@ en:
   isRemote: remote
   maximumInviteeCount: Maximum guest count
   name: Name
-  namePlaceholder: Welcome Party
+  namePlaceholder: Slug Party
   location: Location
   preview: Preview
   previewNoContent: No content to preview 😕
   slug: Slug
   slugPlaceholder: welcome-party
   start: Start
   updated: Updated
-  validationExistenceNone: You've already created an event with id "{slug}"
+  validationExistenceNone: You have already created an event with id "{slug}"
```

Comment:
Do you want to keep it this way?

Reply:
In this specific case "You've already..." sounds too informal to me